packages/legacy-workbench/src/css/lib/ontotext-yasgui-web-component.css

```diff
@@ -49,46 +49,71 @@ yasgui-component .yasgui-toolbar .btn-orientation {
     background-color: transparent !important;
 }
 
-yasgui-component .yasqe .CodeMirror {
-    font-family: var(--mono-font) !important;
-    font-size: 16px;
-    background-color: var(--gw-editor-content-background) !important;
-    color: var(--gw-editor-content-color) !important;
+/*----------------------------------------*/
+/*     Overrides for the moxer theme      */
+/*----------------------------------------*/
+.yasgui-host-element .yasgui .cm-s-moxer.CodeMirror {
+    border-color: var(--gw-tabs-tab-active-border-color);
+    background-color: #1C1C1C;
 }
 
-yasgui-component .yasqe .CodeMirror-gutters {
-    color: var(--gw-editor-toolbar-numbers-color);
-    background-color: var(--gw-editor-toolbar-numbers-background);
+.cm-s-moxer .CodeMirror-linenumber {
+    color: #777777;
 }
 
-.yasgui-host-element .yasgui .CodeMirror,
-yasgui-component .yasqe .CodeMirror-cursor,
-yasgui-component .yasqe .CodeMirror-gutters {
-    border-color: var(--gw-editor-toolbar-border-color);
+yasgui-component .yasgui .yasqe .cm-s-moxer.CodeMirror .yasqe_buttons .custom-button,
+yasgui-component .yasqe .cm-s-moxer.CodeMirror .yasqe_buttons {
+    background-color: #1C1C1C !important;
```

Comment:
Why all these `!important` flags are needed here? Aren't your selectors more specific here?

Reply:
It overrides another selector that uses `!important`